libpromises/pipes.c

```diff
@@ -119,7 +119,7 @@ Rlist *PipeReadData(const IOData *io, int pipe_timeout_secs, int pipe_terminatio
     }
 
     char *read_string = BufferClose(data);
-    Rlist *response_lines = RlistFromSplitString(read_string, '\n');
+    Rlist *response_lines = RlistFromStringSplitLines(read_string);
```

Comment:
I would prefer it if `crlf` was a boolean argument to this functions, so you can test both modes on unix platforms.